pages/testimony.tsx

```diff
@@ -1,15 +1,32 @@
 import { useRouter } from "next/router"
-import { Button, Spinner } from "react-bootstrap"
+import { Col, Row, Spinner } from "react-bootstrap"
 import { Container } from "../components/bootstrap"
 import {
   useBill,
   usePublicProfile,
   usePublishedTestimonyListing
 } from "../components/db"
 import { formatBillId } from "../components/formatting"
-import { Wrap } from "../components/links"
 import { createPage } from "../components/page"
 import { ViewAttachment } from "../components/ViewAttachment"
+import styled from "styled-components"
+
+const PositionSentence = styled(Container)`
+  background-color: white;
+  border-radius: 0.75rem;
+  padding: 1rem;
+  text-align: center;
+  font-weight: bold;
+`
+const Testimony = styled(Container)`
+  background-color: white;
+  border-radius: 0.75rem;
+  padding: 1rem;
+  background-image: url("quote-left.svg");
```

Comment:
let's get rid of the quote here, adjust the padding, and use `font-family: "Nunito";`